src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -117,449 +117,466 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="Action_Consolidate" xml:space="preserve">
-    <value>Consolidate</value>
+  <data name="Deprecation_LinkTooltip" xml:space="preserve">
```

Comment:
What happened here? :D 

It seems like the alphabetical order is gone.

I just want to make sure that the next commit here doesn't shift everything around again.

Reply:
I added the warning png and it changed the ordering like this, from what I see other Resources.resx are also in alphabetical order. I'll double check this